mcodingbot/plugins/python.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+import crescent
+import hikari
+
+from mcodingbot.utils import Plugin
+
+plugin = Plugin()
+
+PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
+
+
+def get_pep_link(pep_number: int) -> str:
+    return f"<https://peps.python.org/pep-{pep_number:04}/>"
+
+
+@plugin.include
+@crescent.command(name="pep", description="Find a Python Enhancement Proposal")
+class PEPCommand:
+    pep_number = crescent.option(int, "the PEP number")
+
+    async def callback(self, ctx: crescent.Context) -> None:
+        await ctx.respond(get_pep_link(self.pep_number))
+
+
+@plugin.include
+@crescent.event
+async def on_message(event: hikari.MessageCreateEvent) -> None:
+    if event.message.content is None or event.author.is_bot:
```

Comment:
might as well do `if not event.message.content`, as it's pointless to check empty strings anyways